gtfs-realtime/spec/en/service-alerts.md

```diff
@@ -24,6 +24,8 @@ Entities are selected using their GTFS identifiers, and you can select any of th
 *   Trip - affects a particular trip
 *   Stop - affects a particular stop
 
+You may include multiple entity selectors. When multiple entities are selected in one Alert feed entity, they should be interpreted as being joined by the `AND` logical operator. If you would like to join the entities by the `OR` operator, you should include them in separate Alert feed entities.
```

Comment:
>If you would like to join the entities by the `OR` operator, you should include them in separate Alert feed entities.

 -> 

>If you would like to represent an alert that affects more than one entity (e.g., an entire route plus a stop not on that route), you should add multiple `informed_entity` clauses to your `alert`, with each of them applying to the affected entity (e.g., one `informed_entity` for the affected route, and one `informed_entity` for the affected stop not on that route).

We can include multiple `informed_entity` clauses, so we can have the same Alert apply to multiple distinct things without having to duplicate the entire alert in the feed.

Reply:
I changed this paragraph to this:

You may include multiple entity selectors in one `informed_entity`. When multiple fields are selected in one `informed_entity`, they should be interpreted as being joined by the `AND` logical operator. In other words, the alert should only be applied in a context that meets all of the fields provided in an `informed_entity`. For example, if `route_id: "1"` and `stop_id: "5"` are both included in one `informed_entity`, then the alert should apply only to route 1 at stop 5.  It should NOT be applied to any other stop on route 1, and it should NOT be applied to any other route at stop 5.

If you would like to represent an alert that affects more than one entity (e.g. an alert to all stops along route 1 and to all routes that visit stop 5) , you should add multiple `informed_entity` to your `alert`, with each of them applying to the affected entity (e.g. one `informed_entity` that includes route 1 and another `informed_entity` that includes stop 5).

Does this look good?